src/inmanta/plugins.py

```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
```

Comment:
Do we want just `abc` or `collection.abc`

Reply:
I don't have a strong opinion on this.